### PR TITLE
add jmespath to requirements

### LINKED ISSUE #63: [RFE]  add python2-jmespath to dependencies
Minimal container installs require python2-jmespath

```
TASK [oVirt.vm-infra : Apply tags from VM profiles] **********************************************************************************************
task path: /usr/share/ansible/roles/ovirt.vm-infra/tasks/vm_state_present.yml:112
fatal: [localhost]: FAILED! => {
    "msg": "You need to install \"jmespath\" prior to running json_query filter"
}
```

### PR DESCRIPTION
**Pull request by mnecas**
Fixes https://github.com/oVirt/ovirt-ansible-image-template/issues/63
@mwperina

### CODE REVIEW

**Comment by mnecas** [comment]
ci build please

File: README.md [hunk 1]
```diff
@@ -50,6 +50,7 @@ Requirements
 
  * Ansible version 2.9 or higher
  * Python SDK version 4.3 or higher
+ * python3-jmespath
```
Comment (mwperina) [inline README.md]: Isn't it worth to mention also python2 variant?

File: ovirt-ansible-vm-infra.spec.in [hunk 2]
```diff
@@ -14,7 +14,7 @@ Group:          Virtualization/Management
 BuildArch:      noarch
 Url:            http://www.ovirt.org
 
-Requires: ansible >= 2.9.0
+Requires: ansible >= 2.9.0, python3-jmespath
```
Comment (mwperina) [inline ovirt-ansible-vm-infra.spec.in]: Please put python3-jmespath on separate line and IMO it's worth also to split it to support also EL7:

```
%if 0%{?rhel} <= 7
Requires	python-jmespath
%else
Requires	python3-jmespath
%endif
```

**Review by mwperina — CHANGES_REQUESTED** [review]

**Review by mwperina — APPROVED** [review]
+1

**Pull request merged**